tests/5.1/masked/test_masked.F90

```diff
@@ -0,0 +1,54 @@
+!/===--- test_masked.F90 -------------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! This test utilizes the masked construct to ensure that a section of code
+! only runs on a certain thread. When the filter clause is not present, the
+! primary thread runs the structured block.
+!
+!/===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_masked
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
+  OMPVV_TEST_OFFLOADING
+
+  OMPVV_TEST_VERBOSE(masked() .ne. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION masked()
+    INTEGER:: errors = 0
+    INTEGER:: total = 10
+    INTEGER:: ct = 0
+    INTEGER, DIMENSION(N):: x
+    INTEGER:: i
+
+    OMPVV_INFOMSG("test_masked")
+
+    DO i = 0, N
+       x(i) = 0
```

Comment:
```
Warning: Array reference at (1) out of bounds (0 < 1) in loop beginning at (2)
```
Thus, either change the bounds to `DIMENSION(0:N)` or the loop beginning to `1`.

Additionally, `x` assigned to outside any OpenMP construct but otherwise not used. (I think @spophale did remark this already.)